packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -49,6 +49,7 @@ const CFDsListing = () => {
         showTopUpModal,
         no_CR_account,
         setSelectedAccount,
+        financial_restricted_countries,
```

Comment:
let's call it with different name so we cover derivX as well
ex: CFDs_restricted_countries